catalogue/webapp/pages/search/images.tsx

```diff
@@ -177,12 +195,29 @@ ImagesSearchPage.getLayout = getSearchLayout;
 export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
   async context => {
     const serverData = await getServerData(context);
+    const query = context.query;
 
     if (!serverData.toggles.searchPage) {
       return { notFound: true };
     }
 
     const params = fromQuery(context.query);
+
+    // Stop here if no query has been entered
+    if (!params.query) {
+      return {
+        props: removeUndefinedProps({
+          imagesRouteProps: params,
+          serverData,
+          query,
+          pageview: {
+            name: 'images',
+            properties: { totalResults: 0 },
```

Comment:
"I entered a search query and I got no results" and "I didn't enter a search query" are two different events, can we track them differently?

Reply:
[Started a thread](https://wellcome.slack.com/archives/C02ANCYL90E/p1669717732492449) for @harrisonpim but he's not in today so I'll treat this separately.